rootston/desktop.c

```diff
@@ -120,19 +120,48 @@ static void view_update_output(const struct roots_view *view,
 	}
 }
 
+void view_update_cursors(const struct roots_view *view,
+		const struct wlr_box *before) {
+	struct roots_desktop *desktop = view->desktop;
+
+	if (view->wlr_surface == NULL) {
+		return;
+	}
+
+	struct wlr_box box;
+	view_get_deco_box(view, &box);
+
+	struct roots_seat *seat;
+	wl_list_for_each(seat, &desktop->server->input->seats, link) {
+		if (seat->cursor->mode != ROOTS_CURSOR_PASSTHROUGH) {
+			continue;
+		}
+		bool in_before = before != NULL && wlr_box_contains_point(before,
+			seat->cursor->cursor->x, seat->cursor->cursor->y);
+		bool in_now = wlr_box_contains_point(&box,
+			seat->cursor->cursor->x, seat->cursor->cursor->y);
+		if (before) {
+                       wlr_log(L_ERROR, "in_before: %d, %fx%f ∈ %dx%d+%dx%d\n"
```

Comment:
Need tabs instead of spaces here and 2 lines below.

Reply:
Bleh, didn't mean to push that at all, sorry. will force push in a min.